prebuild/Dockerfile

```diff
@@ -7,6 +7,8 @@ USER root
 ARG OPENCOARRAYS_VERSION=1.9.1
 ENV GCC_VERSION 7.1.0
 
+ENV NUM_IMAGES 4
+
```

Comment:
Can you please move this to right above `USER ${NB_USER}` at the bottom of the file? It impacts which layers need to get rebuilt, and we want it near the bottom.